.github/actions/integration/entrypoint.sh

```diff
@@ -39,4 +39,23 @@ then
   echo "::endgroup::"
 fi
 
+
+if [ $2 -eq 3 ]
```

Comment:
Are the constants in bash by any chance? :)

Reply:
I'll add named value so it's clearer